index.js

```diff
@@ -0,0 +1,26 @@
+const express = require("express");
+const app = express();
+const port = process.env.PORT || 4000;
+
+// const db = require("./db");
+// console.log("SEE THIS", db);
+
+const cors = require("cors");
+const corsMiddleWare = cors();
+app.use(corsMiddleWare);
+
+const parserMiddleWare = express.json();
+app.use(parserMiddleWare);
+
+const userRouter = require("./User/router");
+app.use(userRouter);
+
+const videosRouter = require("./Videos/router");
+app.use(videosRouter);
+
+const quotesRouter = require("./Quotes/router");
```

Comment:
better to do all imports at the top of the file, something like:
1. Libraries and modules
2. All my files (routers, models, middlewares)
 and then do
`const app = express()` etc etc